scripts/credhub_shell.sh

```diff
@@ -12,10 +12,11 @@ function cleanup () {
 }
 trap cleanup EXIT
 
+BOSH_CA_CERT="$(aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/bosh-CA.crt" -)"
 BOSH_IP=$(aws ec2 describe-instances \
```

Comment:
Do you actually still need BOSH_IP? Or can you just use `bosh-external.${SYSTEM_DNS_ZONE_NAME}` instead?

Reply:
9f4b313

Reply:
Good shout